docs/guide-using-an-ide-for-backend.md

```diff
@@ -172,38 +187,13 @@ Sadly, it's not possible to generate multiple files from one template, so the Me
 * Save the configuration
 * In the tool bar, select the `molgenis-app [exploded]` configuration and press the play button.
 
-This'll build and deploy molgenis to tomcat and open it in the browser.
-Whenever you tab from molgenis to the browser, all modified resources will be copied to the deployed exploded war.
+This'll build and deploy MOLGENIS to tomcat and open it in the browser.
```

Comment:
IntelliJ automatically opens `localhost:8080`, can we configure it so that it opens the correct url?

Reply:
Yes you can. Change the url of the browser to localhost instead of localhost:8080. Be sure to point tomcat to 8080.